components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -348,45 +353,45 @@ end
 
 function CustomLeague:_getMaps(args)
 	local maps = {}
```

Comment:
```suggestion
	if _league.maps then
		return _league.maps
	end

	local maps = {}
```
If the result should be cached (seems a bit overkill for this function however). If no cache, remove L379. If cached, remove the input arg (and use _league.args).

Reply:
I thought caching might be nice due to it being called thrice, but if you say it's fine i'll remove it.

Reply:
Caching is fine, but I don't like the current method (where the invocation order is assumed)